starlark/value.go

```diff
@@ -276,7 +274,7 @@ type HasSetKey interface {
 var _ HasSetKey = (*Dict)(nil)
 
 // A HasBinary value may be used as either operand of these binary operators:
-//     +   -   *   /   //   %   in   not in   |   &   ^   <<   >>
+// +   -   *   /   //   %   in   not in   |   &   ^   <<   >>
```

Comment:
`//   + ...` will get formatted to `//  - ...` with the new go1.19 doc strings: https://tip.golang.org/doc/comment. To keep the plus I removed the padding from the start of the line. Is there a better way to solve this? Can remove doc string changes too.

Reply:
I'm inclined to leave it alone. The source is clear and even if the formatting loses one operator the meaning remains obvious.

Reply:
This looks fine. I don't see a perfect solution.